doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,223 @@
+
+<!-- TODO: Once http://shotgun/detail/Ticket/12251 is resolved, add note explaining that Box nodes and custom-authored shaders don't support enabling/disabling without some additional setup. -->
+
+```eval_rst
+===================================== =============================================
+Action			                      Control or Shortcut
+===================================== =============================================
+Cut node(s)                           :kbd:`Ctrl` + :kbd:`X`
+Copy node(s)                          :kbd:`Ctrl` + :kbd:`C`
+Paste node(s)                         :kbd:`Ctrl` + :kbd:`V`
+Delete node(s)                        :kbd:`Backspace`
+                                      
+                                      or
+                                      
+                                      :kbd:`Delete`
+Enable/disable node(s)                :kbd:`D`
+===================================== =============================================
+```
+
+
+### Node Connections and Layout ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Control or Shortcut
+===================================== =============================================
+Connect plug                          Click and drag plug to another plug
+Disconnect plug                       Click and drag connection to background
+Insert node at connection             Drag node onto connection
+Auto-arrange selected nodes           :kbd:`Ctrl` + :kbd:`L`
+===================================== =============================================
+```
+
+
+### Node Bookmarks ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Control or Shortcut
+===================================== =============================================
+Bookmark node                         Right-click node > *Bookmark*
+Connect to bookmarked node            Right-click plug > *Connect Bookmark* > select
+                                      node
+===================================== =============================================
+```
+
+
+## _Viewer_ ##
+
+> Note :
+> For the following controls and shortcuts, the cursor must hover over the _Viewer_.
+
+
+### General Controls ###
+
+```eval_rst
+===================================== =============================================
+Action			                      Control or Shortcut
+===================================== =============================================
+Pan                                   :kbd:`Alt` + click and drag
+Zoom/dolly                            :kbd:`Alt` + right-click and drag
+                                      
+                                      or
+                                      
+                                      Mouse wheel up/down
+Frame view to contents                :kbd:`F`
+Toggle red channel                    :kbd:`R`
```

Comment:
now that we separated out 3d / 2d into their own tables, these channel toggles are 2d only controls.

Reply:
Thanks. Fixed.